nix/modules/fedimintd.nix

```diff
@@ -173,60 +199,102 @@ in
         eachFedimintd);
 
 
-    systemd.services = mapAttrs'
+    systemd.services =
+      mapAttrs'
+        (fedimintdName: cfg: (
+          nameValuePair "fedimintd-${fedimintdName}" (
+            let
+              startScript = pkgs.writeShellScript "fedimintd-start" (
+                (if cfg.bitcoin.rpc.secretFile != null then
+                  ''
+                    secret=$(${pkgs.coreutils}/bin/head -n 1 "${cfg.bitcoin.rpc.secretFile}")
```

Comment:
Which user does this start script run as? Do we need to worry about permissions? Would be nice to document if so.

Reply:
It runs as the unit's user and group which is configurable and default to `fedimint-<name>`